solutions/swb-reference/integration-tests/support/resources/datasets/dataset.ts

```diff
@@ -17,6 +17,15 @@ export default class Dataset extends Resource {
     return this._axiosInstance.post(`${this._api}/share`, requestBody);
   }
 
+  public async associateWithProject(
```

Comment:
I wanted to use the API endpoint that users will actually call.

Reply:
Environment Type does that too in a more complicated way. I am good with this but wanted to point out there was another way to do it in case you hadn't seen it!